source/assets/styles/index.css

```diff
@@ -162,14 +162,17 @@ main .chat-messages {
   cursor: pointer;
   margin: 0.25rem 0;
 } */
-
+aside {
+  height: 92vh; 
```

Comment:
try another one but not vh (only by itself), cause it will cause a lot of trouble when resizing the windows